config/owasp-supressions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress>
+    <notes><![CDATA[file name: Ehcache modules are not Gradle!]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.ehcache.*@.*$</packageUrl>
+    <cve>CVE-2019-11065</cve>
```

Comment:
I understand that this loophole existed earlier in gradle , curious if there is any other reason to add this.

Reply:
I don't get why you added this suppression file at all, it's certainly worth a comment. Also, doesn't it miss a license header?

Reply:
I'll add a comment to explain the exclusion.